warehouse/query-core/src/main/java/datawave/query/attributes/Attribute.java

```diff
@@ -59,6 +61,18 @@ public ColumnVisibility getColumnVisibility() {
         return Constants.EMPTY_VISIBILITY;
     }
 
+    /**
+     * Some callers only require the raw bytes
+     *
+     * @return the byte array that backs the column visibility
+     */
+    public byte[] getColumnVisibilityBytes() {
+        if (isMetadataSet()) {
+            return metadata.getColumnVisibilityData().getBackingArray();
```

Comment:
I think we want `toArray()` here. I'm pretty sure we've hit issues before where the internal byte[] is reused so you'd need to account for the starting offset (which `toArray()` does).